app/controllers/v1/submissions_controller.rb

```diff
@@ -62,10 +52,6 @@ def complete_submission!(submission)
     SubmissionCompletion.new(submission).perform!
   end
 
-  def create_submission_params
-    params.require(:submission).permit(:task_id)
-  end
-
```

Comment:
As discussed, we've removed the strong param here because they're only really useful in mass assignment (which we aren't using in this context, talking as we do directly to the API). We expect the `update_submission_params` to also go away in due course.